IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/MSButton.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MSButton.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/20/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable public class MSButton: UIButton {
```

Comment:
A couple of questions on this class:

- If I change border color or width AFTER the button has been drawn, it won't get updated, right?
- Are customizations visible on the IB? (I think you need to handle them a bit different to achieve that, but I'm not sure)